kernel/dune

```diff
@@ -3,14 +3,19 @@
  (synopsis "The Coq Kernel")
  (public_name coq.kernel)
  (wrapped false)
- (modules_without_implementation cinstr nativeinstr)
+ (modules (:standard \ uint63_x86 uint63_amd64 write_uint63))
  (libraries clib config lib byterun))
 
 (rule
  (targets copcodes.ml)
  (deps (:h-file byterun/coq_instruct.h) make-opcodes)
  (action (run ./make_opcodes.sh %{h-file} %{targets})))
 
+(rule
+ (targets uint63.ml)
+ (deps write_uint63.ml uint63_x86.ml uint63_amd64.ml)
+ (action (run %{ocaml} unix.cma write_uint63.ml)))
```

Comment:
It is usually more convenient to define a private executable and then have it `run` to generate the file.

Even if more verbose, things such as `unix` will be handled in a more principled way.

Reply:
Note that `(run foo)` automatically adds a dependency on `foo` if in scope. It tends to be more portable too.

Reply:
@vbgl can you have a look at this?